model/src/main/kotlin/OrtResult.kt

```diff
@@ -567,6 +567,12 @@ data class OrtResult(
             isPackageExcluded(id)
         }
 
+    /**
+     * Return `true` if and only if the given [issue] is excluded in scope of the given [id].
```

Comment:
Can you better describe what "in scope of the given id" means? IIUC it's either that the id is excluded completely or that the `affectedPath` of the issue matches a path exclude configured for id? The `excludedAffectedPathIssuesForId` property could use some documentation. And maybe also a renaming to make the purpose clearer, something like `issuesWithExcludedAffectedPathById`.